test/python/transpiler/test_basis_translator.py

```diff
@@ -610,17 +610,19 @@ def test_cx_bell_to_cz(self):
 
         qr = QuantumRegister(2, 'q')
         expected = QuantumCircuit(qr)
-        expected.rz(pi, qr)
+        expected.rz(0, qr)
         expected.rx(pi / 2, qr)
         expected.rz(3 * pi / 2, qr)
         expected.rx(pi / 2, qr)
         expected.rz(3 * pi, qr)
+        expected.rx(pi, qr)
         expected.cz(qr[0], qr[1])
-        expected.rz(pi, qr[1])
+        expected.rz(0, qr[1])
         expected.rx(pi / 2, qr[1])
         expected.rz(3 * pi / 2, qr[1])
         expected.rx(pi / 2, qr[1])
         expected.rz(3 * pi, qr[1])
+        expected.rx(pi, qr[1])
```

Comment:
With the additional equivalences this decomposition gets two more RX gates than before, do you have an idea why that happens?